src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
```

Comment:
we could separate the classes into some packages like `model`, `service`, `supplier`, but that is up to you